src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/PostsListPage.java

```diff
@@ -75,18 +73,9 @@
   private final CategoriesFieldset categories = new CategoriesFieldset();
 
 
-  public PostsListPage open(String wikiID) {
-    driver.get(urlBuilder.getUrlForWiki() + String.format(PATH, wikiID));
```

Comment:
turns out, the way it was implemented, it never worked - accessing some wiki url followed by `/d/f/wikiID` had no effect, and redirected to `/d/f/`